### DISCUSSION TITLE
MOT 2020 Data

### DISCUSSION THREAD

**Discussion by diesendruck** [ethvis/qd-track#7] [General] [answered]
From the BDD site:
https://bdd-data.berkeley.edu/portal.html#download

Clicking on "MOT 2020 Data" goes to:
http://dl.yf.io/bdd100k/tracking20/

There are .zip and .zip.md5. For example:
```
images20-track-test-1.zip                          18-Apr-2020 01:10      4G
images20-track-test-1.zip.md5                      22-Apr-2020 07:19      32
```

Are they the same content? And does the model need both?

**Comment by fyu** [comment, marked as answer]
You only need the zip files. As the names suggested, the `.md5` files can help you check the integrity of the downloads by matching the md5 code of the zip files.
